### PR TITLE
feat: add docs about overriding the to address for emails

### PR DESCRIPTION
**Pull request by cjbell**
### Description

Adds docs for overriding the `to` address for all emails

### Tasks

KNO-7458

### CODE REVIEW

File: content/integrations/email/settings.mdx [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
 layout: integrations
-tags: ["bcc", "cc", "JSON overrides"]
+tags: ["bcc", "cc", "JSON overrides", "email settings", "email overrides"]
 ---
 
-In this guide we cover how to configure email channels in Knock.
-
-## An intro to email channel settings
-
 Knock email channel configurations support a number of settings. These include include email-specific fields (such as cc, bcc, and reply-to) as well as JSON overrides to be pass in API calls to the configured provider's API endpoints.
```
Comment (cellomatt) [inline content/integrations/email/settings.mdx:9]:
```suggestion
Knock email channel configurations support a number of settings. These include email-specific fields (such as `cc`, `bcc`, and `reply-to`) as well as JSON overrides to be passed in API calls to the configured provider's API endpoints.
```
Not your change, but these should be fixed 😅 

Not sure about `code` style on these fields but if we add here we should be consistent below as well

File: content/integrations/email/settings.mdx [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
+When you configure a setting in a channel's configuration, it will be used on all instances of that channel across all workflows.
 
-### Configuring email settings
+## Configuring email settings
 
-The cc, bcc, and reply-to fields can be found at the bottom of the channel environment configuration modal's "Settings" tab.
-The JSON overrides field can be found under the "Overrides" Tab.
+You can override email settings on a per-channel basis, or on a per-template basis.
+
+- **At the channel-level**. The to, cc, bcc, and reply-to fields can be found in the "Overrides" section of the channel's "Settings" tab.
```
Comment (cellomatt) [inline content/integrations/email/settings.mdx:17]:
```suggestion
- **At the channel level**. The `to`, `cc`, `bcc`, and `reply-to` fields can be found in the "Overrides" section of the channel's "Settings" tab.
```
nit, don't think hyphen is necessary. also added `code` style to fields

File: content/integrations/email/settings.mdx [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
+- **At the template-level**. You'll find the email settings under the "Manage template settings" button.
```
Comment (cellomatt) [inline content/integrations/email/settings.mdx:18]:
```suggestion
- **At the template level**. You'll find the email settings under the "Manage template settings" button.
```
nit

**Review by meryldakin — APPROVED** [review]

File: content/integrations/email/settings.mdx [hunk 4]
```diff
@@ -27,7 +25,13 @@ As an example, if you wanted to conditionally change the "From name" on an email
 {{ actor.from_name | default: "no-reply@knock.app" }}
 ```
 
-### CC and BCC fields
+## Overriding the default `to` address
+
+By default Knock will send your emails to the `email` property stored on the `recipient` for the workflow run. If you need to override this, you can do so by setting the `to` field in your email configuration either at the channel or the template level.
+
+As an example, if you wanted to send all emails to a single address, you could set the `to` field to that address at the channel to either a static value (like `hello@example.com`) or a dynamic value (like `{{ data.email_to_override }}`).
```
Comment (cellomatt) [inline content/integrations/email/settings.mdx:32]:
```suggestion
As an example, if you wanted to send all emails to a single address, you could set the `to` field at the channel level to either a static value (like `hello@example.com`) or a dynamic value (like `{{ data.email_to_override }}`).
```

File: content/integrations/email/settings.mdx [hunk 5]
```diff
@@ -27,7 +25,13 @@ As an example, if you wanted to conditionally change the "From name" on an email
...
+
+## Setting `cc` and `bcc` addresses
 
 The cc and bcc fields can be used to support a single or multiple addresses. In order to use several addresses on any of these fields, make sure to separate them with a comma.
```
Comment (cellomatt) [inline content/integrations/email/settings.mdx:36]:
```suggestion
The `cc` and `bcc` fields can be used to support a single or multiple addresses. In order to use several addresses on any of these fields, make sure to separate them with a comma.
```
Just for consistency

**Review by cellomatt — APPROVED** [review]
Left a few suggestions

**Pull request merged**